class/actions_doc2project.class.php

```diff
@@ -327,5 +327,28 @@ function doActions($parameters, &$object, &$action, $hookmanager)
 		return 0;
 	}
 
-
+	/**
+	 * afterCreateProject
+	 *
+	 * @param array()		   $parameters	  Hook metadatas (context, etc...)
+	 * @param CommonObject    &$object        The object being processed (e.g., an invoice, proposal, etc...)
+	 * @param string          &$action        The current action (usually create, edit, or null)
+	 * @param HookManager      $hookmanager   Hook manager instance to allow calling another hook
+	 * @return int                            Returns < 0 on error, 0 on success, 1 to replace standard code
+	 */
+	function afterCreateProject($parameters, &$object, &$action, $hookmanager): int
+	{
+		global $conf, $langs, $db, $user;
+		if ($action == 'afterCreateProject' && $conf->global->DOC2PROJECT_ADD_USAGE_TASK_ON_PROJECT){
```

Comment:
tu as des warnings en php 8 si ton élément n'est pas setté
```suggestion
		if ($action == 'afterCreateProject' && !empty($conf->global->DOC2PROJECT_ADD_USAGE_TASK_ON_PROJECT)){
```

Reply:
dans ton changelog tu met "lors de la création automatique d'un projet", est-ce que tu es sur que l'action afterCreateProject est utilisé uniquement pour la création automatique ? Ca va pas cocher tous le temps la case ?